DuckDuckGo/Autoconsent/AutoconsentUserScript.swift

```diff
@@ -291,13 +290,44 @@ extension AutoconsentUserScript {
         }
     }
 
+    @MainActor
+    func handlePopupFound(message: WKScriptMessage, replyHandler: @escaping (Any?, String?) -> Void) {
+        guard let messageData: PopupFoundMessage = decodeMessageBody(from: message.body) else {
+            replyHandler(nil, "cannot decode message")
+            return
+        }
+        Logger.autoconsent.info("Cookie popup found: \(String(describing: messageData))")
+        guard let url = URL(string: messageData.url),
+              let host = url.host else {
+            replyHandler(nil, "cannot decode message")
+            return
+        }
+
+        if messageData.cmp == "filterList" {
```

Comment:
Can this be moved to the `Constants` section?
Also please add some documentation comments what the "filterList" means and when it‘s supposed to be included set as `messageData.cmp` value - it‘s now unclear; Also please add a comment what `messageData.cmp` means in the MessageData structs?